packages/builder/src/schema.d.ts

```diff
@@ -10,6 +10,7 @@ export interface Schema extends JsonObject {
   fix: boolean;
   cache: boolean;
   cacheLocation: string | null;
+  cacheStrategy: 'content' | 'metadata' | null;
```

Comment:
Shouldn't `null` be `undefined` here? Same as `cacheLocation` above.

Reply:
The problem is that `Schema` extends [`JsonObject`](https://github.com/angular/angular-cli/blob/37a06a7c37f5b4286d58b475e6e12c86f00fac5b/packages/angular_devkit/core/src/json/interface.ts#L57-L59), which doesn't accept `undefined`.

Reply:
Can optional property `?:` be used?

Reply:
No, because it has the same effect of `| undefined`.

Reply:
I mean add it into `JsonObject`.